feather/server/src/client.rs

```diff
@@ -547,6 +547,16 @@ impl Client {
         });
     }
 
+    pub fn send_entity_metadata(&self, network_id: NetworkId, metadata: EntityMetadata) {
+        if self.network_id().unwrap() == network_id {
```

Comment:
Will this unwrap ever panic? If not, add a comment asserting why it's safe to unwrap here.

Reply:
Not calling the function anymore.

Reply:
omg derp I never thought about that or I would have suggested that directly